sanity-io/structured-content-2022
# Redesign Speakers section

# Redesign Speakers section

## Intent

Implement new design for the Speakers section based on the Figma sketches for the Speakers page, as described in Shortcut story [Speakers page: Front-end](https://app.shortcut.com/sanity-io/story/14186/speakers-page-front-end).

## Description

As discussed in standup, we'll be using the existing "Pages"/"Routes" machinery in Sanity to define the speakers index page. For efficiency and consistency I changed the styling of the Section page completely, rather than forking it into two implementations for the front page and speakers page separately, since they will have the same styling eventually anyway.

The heading and CTA is not covered by the Figma sketches, but I made some select adjustments of the existing styling and Sindre has looked it over.

There is a problem with the "hotspot", but this is not new to this PR. I filed https://app.shortcut.com/sanity-io/story/17751/investigate-and-if-possible-fix-problem-with-cropping-of-speaker-photos now.

## Testing this PR

1. Open https://structured-content-2022-web-git-redesign-speakers-section.sanity.build/speakers, verify that the speakers are alphabetized on name and are presented with a layout/styling as indicated in the Figma sketches linked above (except some of the speakers don't have an image and the Figma sketch doesn't cover that edge case)
2. Open https://structured-content-2022-web-git-redesign-speakers-section.sanity.build/ and scroll down to "Featured speakers", verify that the featured speakers (Carrie Hane and Simen Svale Skogsrud) are presented in the same way
